receiver/awscloudwatchreceiver/testdata/processed/prefixed.json

```diff
@@ -29,7 +29,7 @@
                     "logRecords": [
                         {
                             "timeUnixNano": "1665166251014000000",
-                            "observedTimeUnixNano": "1665166998995360000",
+                            "observedTimeUnixNano": "1673025345210224000",
```

Comment:
Is this change necessary if observed timestamp is ignored?

Reply:
It's not. Thanks for catching